Stripe/STPPaymentMethodTuple.m

```diff
@@ -27,22 +27,23 @@ + (instancetype)tupleWithPaymentMethods:(NSArray<id<STPPaymentMethod>> *)payment
     return tuple;
 }
 
-+ (instancetype)tupleWithCardTuple:(STPCardTuple *)cardTuple
-                   applePayEnabled:(BOOL)applePayEnabled {
-    NSMutableArray *paymentMethods = [NSMutableArray array];
-    for (STPCard *card in cardTuple.cards) {
-        [paymentMethods addObject:card];
-    }
++ (instancetype)tupleWithPaymentMethods:(NSArray<id<STPPaymentMethod>> *)paymentMethods
+                  selectedPaymentMethod:(id<STPPaymentMethod>)selectedPaymentMethod
```

Comment:
nit: can we wrap this file with `NS_ASSUME_NONNULL`s and annotate this as `nullable`?

Reply:
Yep I'll make the change.